code/client/clrcore-v2/BaseScript.cs

```diff
@@ -283,7 +298,90 @@ internal void RegisterKeyMap(string command, string description, string inputMap
 			m_commands.Add(new KeyValuePair<int, DynFunc>(ReferenceFunctionManager.CreateCommand(command, dynFunc, false), dynFunc));
 #endif
 		}
+		
+		#endregion
+		
+		#region NUI Callback registration
+
+		internal void RegisterNuiCallback(string callbackEvent, DynFunc dynFunc)
```

Comment:
Still got some `callbackEvent` names lingering ;) also down below, can you change these to the `callbackName` as well, including all the other occurrences?